web/models.py

```
@@ -799,6 +799,29 @@ class ForumTopic(models.Model):
     created_at = models.DateTimeField(auto_now_add=True)
     updated_at = models.DateTimeField(auto_now=True)
 
+    # Add these methods to the ForumTopic class
+    def upvote_count(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `upvote_count`

(ANN201)